setup.py

```diff
@@ -177,6 +181,14 @@ def get_data_files(name, version, fullname):
     return data_files
 
 
+def debian_has_systemd():
```

Comment:
The use of `debian_has_systemd()` is cleaner than the earlier approach. Thanks for doing the change. 👍